web/tendrl/cluster_expand.rst

```diff
@@ -0,0 +1,70 @@
+Expand cluster
+**************
+
+:authors: 
+          - fbalak@redhat.com
+
+Description
+===========
+
+Cluster expansion as described in `Specification for cluster expansion`_.
+
+Setup
+=====
+
+#. Prepare Gluster cluster.
+
+#. Login user into Tendrl, see :doc:`/web/tendrl/login_positive`
+
+#. Import cluster, see :doc:`/web/tendrl/cluster_import`
+
+#. Prepare another hosts with installed gluster that are going to be used for
+   expansion.
+
+Test Steps
+==========
+
+.. test_action::
+   :step:
+       From one of already imported hosts run:
+       ``gluster peer probe [new_host]``
+       where ``[new_host]`` is host that is going to be used to expand cluster.
+       Repeat for all new hosts.
```

Comment:
I would think about expanding some work with volumes too. E.g. create a new one on all nodes, or expand existing one etc.

Reply:
True, but that could be covered in another test case.